### DISCUSSION TITLE
Aumentar las iteraciones que puede tener una rutina

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#572] [3. Ideas & Feature Requests] [answered]
Hola a todos

Me gustaria saber la posibilidad de aumentar las iteraciones de una rutina (actualmente el máximo es 100). Ya que con el pasar de los tiempos, se están aumentar las etapas y configuraciones de éstas.

Un ejemplo es con el PBiterate, donde dentro de la etapa "step", tengo 7 PB y en el "done" tenga 3 PB. Si debo iterar un campo de 10 elementos, serian 70 iteraciones + 3, sólo para el iterate. En el caso que el campo sea 15, no se completaria de realizar la rutina completa.

Esto nos sucedió con un cliente en donde debemos leer "X" filas de un excel.

Gracias =>

**Comment by acedward** [comment, marked as answer]
Absolutamente. vamos a catalogar esto como un bug.

**Reply by acedward** [reply]
https://github.com/Cotalker/documentation/issues/597

**Reply by NicolasPLG** [reply]
mucha gracias por la gestión @acedward
